relay-event-normalization/src/normalize/span/country_subregion.rs

```diff
@@ -0,0 +1,303 @@
+// country_subregion.rs
+use std::collections::HashMap;
+use std::fmt;
+
+/** Subregions regions as defined by UN M49 standards, the enum value corresponds to the region code */
```

Comment:
This is not a Rust doc comment:

```suggestion
/// Subregions regions as defined by UN M49 standards, the enum value corresponds to the region code.
```

A link to the officially defined mappings or wikipedia entry would be nice.